src/chd.c

```diff
@@ -789,6 +789,13 @@ chd_error cdfl_codec_decompress(void *codec, const uint8_t *src, uint32_t comple
     CODEC INTERFACES
 ***************************************************************************/
 
+#define CHD_MAKE_TAG(a,b,c,d)       (((a) << 24) | ((b) << 16) | ((c) << 8) | (d))
+
+/* general codecs with CD frontend */
+#define CHD_CODEC_NONE 0
+#define CHD_CODEC_CD_ZLIB CHD_MAKE_TAG('c','d','z','l')
+#define CHD_CODEC_CD_LZMA CHD_MAKE_TAG('c','d','l','z')
+#define CHD_CODEC_CD_FLAC CHD_MAKE_TAG('c','d','f','l')
```

Comment:
As my changes got merged, those are in the header file. Only ``CHD_CODEC_NONE`` is missing and should be move there.
https://github.com/rtissera/libchdr/blob/6726f52a1d3ba04cebefade2bfcbb9dfc85eef00/src/chd.h#L207